src/GraphQLSchemaLinter.php

```diff
@@ -172,7 +172,14 @@ protected function parseLinterOutput($path, $err, $stdout, $stderr) {
         }
         */
 
+        if (strlen($stdout) == 65536) {
```

Comment:
Is it possible to query this character limit from somewhere?

Reply:
I looked around for some PHP constants that would represent this. Perhaps there's something in Arcanist.

I used `==` because if the length is greater than it, my assumption that only `65536` characters can be stored in the buffer is no longer true (perhaps an updated version of Arcanist, or a platform difference) and then this check is no longer meaningful. But I think that makes a very good argument for using some Arcanist constant here 🙂